src/core/core.scale.js

```diff
@@ -99,6 +100,26 @@ function computeTextSize(context, tick, font) {
 		context.measureText(tick).width;
 }
 
+function parseFontOptions(options, nestedOpts) {
```

Comment:
We could use `_parseFont` instead:

```javascript
function resolveFontOptions(options, nestedOpts) {
    return helpers.options._parseFont({
        fontColor: valueOrDefault(nestedOpts.fontColor, options.fontColor),
        fontFamily: valueOrDefault(nestedOpts.fontFamily, options.fontFamily),
        fontSize: valueOrDefault(nestedOpts.fontSize, options.fontSize),
        fontStyle: valueOrDefault(nestedOpts.fontStyle, options.fontStyle),
        lineHeight: valueOrDefault(nestedOpts.lineHeight, options.lineHeight)
     });
}
```

Reply:
Done. `fontColor` needed to be separately merged as `_parseFont` doesn't parse it, though.